.gitignore

```diff
@@ -1,3 +1,4 @@
+Gemfile.lock
```

Comment:
I've also observed that excluding `Gemfile.lock` from gem repositories is common.

I just did a quick spot check and it looks like `rails` includes the `Gemfile.lock`, while `rack` and `ominiauth` do not.

One advantage of excluding it is that CI builds will always pull the latest dependencies allowed by the ranges, which can result in finding bugs introduced by new versions of dependencies. On the other hand, not having a `Gemfile.lock` means that you could no longer be testing against an older supported version of a dependency, resulting in possible regressions.

I think that to avoid the latter problem, the best practice I've seen is to have special Gemfile with stricter ranges for testing different (usually major) versions of dependencies. The `omniauth` gem, as an example, has separate Gemfiles for testing `rack` 1.3 and `rack`'s `master` branch in addition to its standard `Gemfile`.